attributes/client.rb

```diff
@@ -23,8 +23,8 @@
 when 'ubuntu'
   default['gluster']['client']['package'] = 'glusterfs-client'
 when 'redhat', 'centos'
-  default['gluster']['client']['package'] = 'glusterfs'
+  default['gluster']['client']['package'] = ['fuse', 'fuse-libs', 'glusterfs', 'glusterfs-fuse']
```

Comment:
The only package that should be needed is `glusterfs-fuse`.  This installs `glusterfs` as a dependency.
Tested on CentOS7

Further info: http://www.unixmen.com/install-glusterfs-server-client-centos-7/